pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -649,6 +667,65 @@ func (adc *attachDetachController) processVolumesInUse(
 	}
 }
 
+// installCRDs creates the specified CustomResourceDefinition for the CSIDrivers object.
+func (adc *attachDetachController) installCRDs() error {
```

Comment:
I am a little concerned about the implications of controller-manager doing schema changes when it starts up. Not claiming it's definitely problematic, just still thinking about it.

Reply:
Ya, we're doing this for alpha because it's the path of least resistance. For beta, I was talking to @timstclair and we were thinking about having a new/dedicated controller that is responsible for installing CRDs (and reinstalling them if they are uninstalled, e.g. like add on manager). Can revisit that for beta.

Reply:
Should we just use addons for alpha? I drafted those changes here: https://github.com/tallclair/kubernetes/commit/a58008ee42b01b7185e1335d2b804dc2df384998

Reply:
> we're doing this for alpha because it's the path of least resistance

That attitude is concerning. Constructing APIs via CRD should not lower our standards for design or make us more willing to ship things we haven't considered the implications of.